Gemfile

```diff
@@ -12,11 +12,13 @@ gem 'devise'
 group :development, :test do
   gem 'pry-byebug'
   gem 'rspec-rails'
+  gem 'actionmailer'
```

Comment:
Actionmailer is apart of rails. You should not have to include it explicitly in your Gemfile